yb-voyager/src/srcdb/postgres.go

```diff
@@ -714,6 +714,10 @@ func (pg *PostgreSQL) CreatePublication(conn *pgconn.PgConn, publicationName str
 	}
 	tablelistQualifiedQuoted := []string{}
 	for _, tableName := range tableList {
+		if leafPartitions[tableName.ForOutput()] != nil {
```

Comment:
We should use utils.StructMap for leafPartitions, not `map[string]...`

Reply:
Yes was thinking of doing it but first thought to fix it. Will change.